### PR TITLE
List seats on homepage

### PR DESCRIPTION
**Pull request by wdehaes**
I deleted the temporary home controller and views. Now the root_path points to the gamescontroller #show action, which loads all games with only one player which users can join. I created the according view (/games/index.html.erb), but did not add any layout. Join buttons are not hooked up yet (and should only work for logged in users. I thought it would be better to do this in a separate PR request.

I also included rspec tests.

### CODE REVIEW

File: spec/controllers/games_controller_spec.rb [hunk 1]
```diff
@@ -25,4 +25,25 @@
...
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should give an overview of games with only one player' do
+      game1 = FactoryGirl.create(:game)
+      game2 = FactoryGirl.create(:game)
+      game3 = FactoryGirl.create(:game)
+      player1 = FactoryGirl.create(:player, game_id: game1.id)
+      player2 = FactoryGirl.create(:player, game_id: game2.id)
```
Comment (wdehaes) [inline spec/controllers/games_controller_spec.rb]: Rubocop give a build error seeing some games players as unused variables. However, player1 seems necessary for the test, and deleting game2, game3, player2 and player3 would in my eyes make the test less convincing for other people reading it. Any suggestions?

Comment (lasley) [reply spec/controllers/games_controller_spec.rb]: Yup it's just saying that you're not using the variables, which is common for a linter - it means you're wasting memory. 

It's standard practice to just not assign the variables that are not used (doesn't mean you can't still create the Factory though). TBH the naming is irrelevant here anyways right? 

If a developer were to look at the code, they would be able to ascertain that one game doesn't have a user, one has one, and one has two. Think of a word problem back in Math class:

```
Mike has five apples.
He sells two apples to Mary & one apple to Joe.
How many apples does Mike have left?
```

In this example, Mike is `game1`. Mary and Joe are 2 & 3, which are irrelevant to the problem at hand - other than the fact that they took apples.

Make sense?

Comment (wdehaes) [reply spec/controllers/games_controller_spec.rb]: Absolutely. I thought rubocop wanted me to delete the lines altogether, but this makes total sense. I'll add a commit to clean it up

**Comment by wdehaes** [comment]
corrected the rubocop issues and merge master

**Review by lasley — APPROVED** [review]
LGTM

**Comment by erinmorrissey** [comment]
LGTM to me too, merging.

**Pull request merged**